lib/common_test/src/ct_snmp.erl

```diff
@@ -107,8 +107,8 @@
 %%% the OTP snmp User's Guide for a list of valid parameters and types). This is 
 %%% done by defining a configuration data variable on the following form:</p>
 %%% <pre>
-%%% {snmp_app, [{manager, [snmp_app_manager_params()]},
-%%%             {agent, [snmp_app_agent_params()]}]}.</pre>
+%%% {snmp_app, [{manager, [SNMPAppManagerParams :: term()]},
```

Comment:
The change on this line and the next will not take effect. Please change it in ct_ftp.xml instead.